RNSentry.podspec

```diff
@@ -17,7 +17,7 @@ Pod::Spec.new do |s|
   s.preserve_paths = '*.js'
 
   s.dependency 'React-Core'
-  s.dependency 'Sentry', '7.30.0'
+  s.dependency 'Sentry/HybridSDK', '7.30.2'
```

Comment:
Add changelog entry for the SDK upgrade.

Reply:
This is an internal change users don't need to know about.
We don't add changelog entries for cocoa when this is the case.
Should I do it here?

Reply:
It's not about the use of HybridSDK header, but rather about the SDK upgrade as per my comment above.
https://github.com/getsentry/sentry-cocoa/releases/tag/7.30.2
There are more fixes in this release that affects this SDK, so it should go under the changelog.